src/main/scala/com/github/mrpowers/spark/fast/tests/ufansi/FansiExtensions.scala

```diff
@@ -0,0 +1,7 @@
+package com.github.mrpowers.spark.fast.tests.ufansi
+object FansiExtensions {
+  implicit class StrOps(c: Seq[Str]) {
```

Comment:
For me it is a part of internal API, so I would suggest mark the implicit class as `private[mrpowers]`?

Reply:
@SemyonSinchenko  Yes, you are right. Done.